src/components/CardEditor/CardGalleryList/CardGalleryList.jsx

```diff
@@ -1,69 +1,92 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {
-  Add16,
-  ChartColumn16,
-  ChartLine16,
-  Table16,
-  SummaryKpi16,
-  Image16,
-} from '@carbon/icons-react';
 
-import { List, Button } from '../../../index';
+import { SimpleList } from '../../../index';
+import { DASHBOARD_EDITOR_CARD_TYPES } from '../../../constants/LayoutConstants';
+
+import timeSeriesImg from './line-graph.svg';
+import simpleBarImg from './simple-bar-graph.svg';
+import groupedBarImg from './bar-grouped-graph.svg';
+import stackedBarImg from './bar-stack-graph.svg';
+import valueImg from './value-kpi.svg';
+import imageImg from './image.svg';
+import tableImg from './data-table.svg';
+import alertTableImg from './alert-table.svg';
+import listImg from './list.svg';
```

Comment:
The build is failing due to these. We can import svg's into stories because storybook uses webpack to load them in. Our current rollup build isn't configured to know what to do with .svg files.

[The core rollup svg plugin](https://github.com/rollup/plugins/tree/master/packages/image/#readme) has a big downside in that it base64 encodes the file so it's going to be larger in size in the bundle than the files are on disk.

Storybook uses svgr, which seems like a better option and has a rollup plugin, [@svgr/rollup](https://www.npmjs.com/package/@svgr/rollup).

Reply:
Okay, I'll take a look at this today

Reply:
Added `svgr` as stated above. Build was successful locally so now testing with Travis